src/main/java/org/broadinstitute/hellbender/utils/recalibration/BaseRecalibrationEngine.java

```diff
@@ -253,10 +257,6 @@ private ReadTransformer makeReadTransform() {
         return f;
     }
 
-    private boolean isLowQualityBase( final GATKRead read, final int offset ) {
-        return read.getBaseQualities()[offset] < recalArgs.PRESERVE_QSCORES_LESS_THAN;
```

Comment:
Ouch, not surprised this one showed up on the profiler, given that `GATKRead.getBaseQualities()` makes a defensive copy on every call...